internal/jstest/jstest.go

```diff
@@ -29,28 +29,46 @@ import (
 )
 
 // Run runs jstests.
-func Run(ctx context.Context, dir string, args, excludeArgs []string) (*internal.TestResults, error) {
+func Run(ctx context.Context, dir string, args []string) (*internal.TestResults, error) {
 	// TODO https://github.com/FerretDB/dance/issues/20
 	_ = ctx
 
 	filesM := make(map[string]struct{})
 
+	excludeFilesM := make(map[string]struct{})
+
 	// remove duplicates if globs match same files
 	for _, f := range args {
+		if strings.HasPrefix(f, "!") {
+			f = strings.TrimPrefix(f, "!")
```

Comment:
There is a new `CutPrefix` that does both operation

Reply:
Crisp!